### PR TITLE
Add test for Group Management

### PR DESCRIPTION
**Pull request by ZitaNemeckova**
TODO: 
- [x] create group
- [x] delete group
- [x] add permissions to a group
- [x] add all permissions to a group
- [x] remove permissions from a group
- [x] add user to a group
- [x] remove user from a group

### CODE REVIEW

**Comment by ZitaNemeckova** [comment]
@hendersonreed please review. Thanks :)

File: test/cypress/integration/group_management.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    });
+
+    it('admin user can add/remove permissions to/from a group', () => {
+        let name = 'testGroup';
+        cy.login(adminUsername, adminPassword);
+        cy.createGroup(name);
+        cy.contains(name).should('exist');
+        cy.addAllPermissions(name);
+        [ 'namespaces', 'collections', 'users', 'groups', 'remotes' ].forEach(permGroup => cy.get(`.pf-l-flex.pf-m-align-items-center.${permGroup}  [placeholder="No permission"]`).should('not.exist'));;
+        cy.removePermissions(name, [{
```
Comment (hendersonreed) [inline test/cypress/integration/group_management.js:38]: Should we just use `removeAllPermissions` here?

**Review by hendersonreed — APPROVED** [review]
Verified that it runs on my machine in both FF and Chromium!

File: test/cypress/integration/group_management.js (hunk 1 above)
Comment (ZitaNemeckova) [reply test/cypress/integration/group_management.js]: Fixed :)

**Comment by hendersonreed** [comment]
Looks good to me! They continue to pass on my machine as well.

**Comment by himdel** [comment]
Backporting because #305 (backported) depends on `createGroup` from this one + https://github.com/ansible/ansible-hub-ui/pull/301 (which conflicts without it).

**Pull request merged**